picard/ui/coverartbox.py

```diff
@@ -255,28 +283,29 @@ def load_remote_image(self, url, mime, data):
         try:
             coverartimage = CoverArtImage(
                 url=url.toString(),
+                types=[u'front'],
                 data=data
             )
         except CoverArtImageError as e:
             log.warning("Can't load image: %s" % unicode(e))
             return
         if isinstance(self.item, Album):
             album = self.item
-            album.metadata.append_image(coverartimage)
+            album.metadata.set_front_image(coverartimage)
```

Comment:
Instead of radio buttons in the context menu, maybe an option in the "user interface" section of the options dialog would be better.